{{cookiecutter.project_dashed}}-src/ansible/group_vars/all

```diff
@@ -1,16 +1,18 @@
-
 use_pvt_dns: false
 
 project_ident: {{ cookiecutter.project_namespace }}
+project_ident_env: {{ cookiecutter.project_namespace }}-{{ '{{' }} app_environment {{'}}'}}
 
-app_db_host: "{{ cookiecutter.database_pvt_host }}"
 {% raw %}
+app_db_host: "{{ lookup('dig', 'sierra.pvt.level12.net')}}"
```

Comment:
I'd use: `dig @8.8.8.8 +short...`

I think you need the +short to work, otherwise there is too much output?
the @8.8.8.8 is b/c some routers cache DNS too long and you can actually tell Google to refresh it's cache for a DNS record.

Reply:
I was going to do anything I would rather we just hit `ns1.linode.com` since that is where the DNS for `level12.net` will be the most accurate. 

I have found, since moving to linode DNS, that the prorogation time is much slower than name.com. It used to be 5-10 mins, it can now take hours.